src/content/reference/helm-chart-values.mdx

```diff
@@ -1395,6 +1395,40 @@ wildcardCertificate:
       key: "ca.crt"
 ```
 
+### regcredsManager
+
+The configuration for the controller manager of the Registry Credentials Operator.
```

Comment:
Could we add a one line description explaining what this component does?
@codyjlandstrom we can link to this doc from the release notes while we don't have a doc explaining the architecture

Reply:
Done https://github.com/okteto/docs/pull/631/commits/09b0b977a65df307438271eb85372728493b0148